build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
         maven { url "https://plugins.gradle.org/m2/" }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.6.3'
+        classpath 'com.android.tools.build:gradle:4.0.0'
```

Comment:
If this is not required for your PR, please keep other things untouched. In case there is a problem, this makes it a lot easier to see where the problem came from.

Reply:
i see. no problem